frontend/src/components/Forms/DetailsForm/index.js

```diff
@@ -9,26 +9,28 @@ const DetailsForm = ({ error, handleChange, name, email }) => {
       headerText="A little bit about you"
       subtitleText="Because you will collect your prize via email"
     >
-      <Input
-        id="name"
-        name="Name"
-        type="text"
-        placeholder="Type in your name"
-        value={name}
-        onChange={handleChange("name")}
-        error={error.hasError}
-        errorMessage={error.message}
-      />
-      <Input
-        id="email"
-        name="Email"
-        type="email"
-        placeholder="Type in your email"
-        value={email}
-        onChange={handleChange("email")}
-        error={error.hasError}
-        errorMessage={error.message}
-      />
+      <>
+        <Input
+          id="name"
+          name="Name"
+          type="text"
+          placeholder="Type in your name"
+          value={name}
+          onChange={handleChange("name")}
+          error={error.hasError}
+          errorMessage={error.message}
+        />
+        <Input
+          id="email"
+          name="Email"
+          type="email"
+          placeholder="Type in your email"
+          value={email}
+          onChange={handleChange("email")}
+          error={error.hasError}
+          errorMessage={error.message}
+        />
+      </>
```

Comment:
Why do we need the fragment here? 🤔

Reply:
I think it's because the `FormLayout` expects `children` to be `children: PropTypes.element.isRequired`

Reply:
`FormLayout` use to accept only one child, so this changed recently I believed.

Reply:
The truth is we don't need errors anymore, but I was thinking of removing errors in another PR so it's not too confusing

Reply:
It's that!

Reply:
What about changing the prop definition to the correct one?

Reply:
I'm very confused with this PR now :sweat:

Reply:
I think what @joaojusto is trying to say is that you should think about what you're expecting `FormLayout` to receive as `children` and adapt the `propTypes` according to that instead of the other way around.